payments/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+
+# version: '4.15.0'
+# services:
+#   db:
+#     image: postgres:15.2
+#     restart: always
+#     environment:
+#       - POSTGRES_USER=tokyobanana
+#       - POSTGRES_PASSWORD=naganohara
+#     volumes:
+#       - db_data:/var/lib/postgresql/data
+#     ports:
+#       - '5432:5432'
+# volumes:
+#   db_data: {}
+
+version: '3'
+
+services:
+  mongo:
+    image: mongo
+    restart: always
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=${MONGO_ROOT_USER}
+      - MONGO_INITDB_ROOT_PASSWORD=${MONGO_ROOT_PASSWORD}
+      - MONGO_INITDB_DATABASE=project
+    ports:
+      - "27017:27017"
+  mongo-express:
+    image: mongo-express
+    restart: always
+    environment:
+      - ME_CONFIG_MONGODB_SERVER=mongo
+      - ME_CONFIG_MONGODB_PORT=27017
+      - ME_CONFIG_MONGODB_ENABLE_ADMIN=true
+      - ME_CONFIG_MONGODB_AUTH_USERNAME=${MONGO_ROOT_USER}
+      - ME_CONFIG_MONGODB_AUTH_PASSWORD=${MONGO_ROOT_PASSWORD}
+      - ME_CONFIG_BASICAUTH_USERNAME=${MONGOEXPRESS_LOGIN}
+      - ME_CONFIG_BASICAUTH_PASSWORD=${MONGOEXPRESS_PASSWORD}
+      - ME_CONFIG_MONGODB_URL=mongodb://${MONGO_ROOT_USER}:${MONGO_ROOT_PASSWORD}@mongodb:27017/
+    depends_on:
+      - mongo
+    ports:
+      - "8888:8081"
+  payments:
+    build: 
+      dockerfile: ./payments.Dockerfile
+    image: payments
+    ports:
+      - "5008:5000"
```

Comment:
I have mapped this to 5008 at the moment so that when we include this in the same docker-compose with our other MS, it will be exposed on a non-conflicting port. let me know if this understanding is incorrect.

Reply:
let's keep it to the standard port 3000 for now, we can make the necessary adjustments during deployment